schemas/context/segmentmembership.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentMembership",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Membership",
+  "type": "object",
+  "description": "Details about a segment membership.",
+  "definitions": {
+    "segmentMembership": {
+      "properties": {
+        "xdm:segmentID": {
+          "title": "Segment ID",
+          "$ref": "https://ns.adobe.com/xdm/context/segmentIdentity",
```

Comment:
Final Edits: All lower case please 'segmentIdentity'->'segmentidentity'. The schema file name is already lower case, so just this ref change.